includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -25,18 +25,33 @@ public function __construct( string $cityId ) {
 	}
 
 	public function getData(): array {
-		$data = [
-			'sitename' => $this->getSiteNameData(),
-			'navigation' => $this->getNavigation()
-		];
+		$globalStateWrapper = new \Wikia\Util\GlobalStateWrapper([
+			'wgCityId' => $this->productInstanceId,
+			'wgDBname' => WikiFactory::IDtoDB($this->productInstanceId)
+		]);
+
+		$data = $globalStateWrapper->wrap( function() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$data". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)